go/plumbing/doc.go

```diff
@@ -0,0 +1,6 @@
+/*
+Package plumbing provides an auto-generated Go client for accessing Netlify's API.
+
+See https://goswagger.io/generate/client.html for more example usage of an API client generated by go-swagger.
+*/
+package plumbing
```

Comment:
memo: this doc and the below example doc/code will be shown in https://pkg.go.dev/github.com/netlify/open-api@v1.4.0/go/plumbing (with new version)